src/sentry/tsdb/base.py

```diff
@@ -77,7 +78,7 @@ class BaseTSDB(object):
     models = TSDBModel
 
     def __init__(self, rollups=settings.SENTRY_TSDB_ROLLUPS):
-        self.rollups = rollups
+        self.rollups = OrderedDict(rollups)
```

Comment:
We look up the sample count by rollup resolution fairly frequently, this simplifies that process.